### PR TITLE
Update zmupdate

### PR DESCRIPTION
**Pull request by connortechnology**
So I broke zmupdate at some point.

This fixes it.

I have been cleaning up all the database connection code, hopefully leaving us with just one place for it.
I am also deprecating the ConfigAdmin module and moving it's two functions into Config.pm.

### CODE REVIEW

**Comment by connortechnology** [comment]
Also of note here are some missing sth->finishes and some cases where we move the statement preparing outside of the loop.

**Comment by knnniggett** [comment]
Rather than specify the full module name like this 
ZoneMinder::Config::loadConfigFromDB();

Would it be more efficient to do something like this?
use ZoneMinder::Config qw(loadConfigFromDB);

my syntax might not be exactly correct.

Actually, I see you added "use ZoneMinder" ...wouldn't that import all zoneminder perl modules, including ZoneMinder::Config ?

File: scripts/zmupdate.pl.in [hunk 1]
```diff
@@ -359,12 +361,12 @@ if ( $interactive ) {
 		if ( $response =~ /^[yY]$/ ) {
 			$dbh->do(q|SET sql_mode='traditional'|); # Elevate warnings to errors
 			print "\nConverting MyISAM tables to InnoDB. Please wait.\n";
+      my $sql = "ALTER TABLE $_ ENGINE = InnoDB";
+      my $sth = $dbh->prepare_cached( $sql ) or die( "Can't prepare '$sql': ".$dbh->errstr() );
```
Comment (knight-of-ni) [inline scripts/zmupdate.pl.in]: Not sure this is right. These two lines got moved outside of the foreach loop just below, which means the default variable $_ won't contain the right value(s).

**Comment by connortechnology** [comment]
Matter of personal preference.  I like to fully specify my function calls. Less ambiguous.

File: scripts/zmupdate.pl.in (hunk 1 above)
Comment (connortechnology) [inline scripts/zmupdate.pl.in]: you are totally right.

**Pull request merged**